src/inc/palclr.h

```diff
@@ -484,7 +484,14 @@
 //  and it provides __declspec(selectany) to instruct the linker to merge
 //  duplicate external const static data copies into one.
 //  
+#if defined(__clang__)
+#define SELECTANY extern
```

Comment:
The clang option `-fms-extensions` enables this `__declspec`. We use it for Unix builds. As for your question on the right test to detect a windows build, you can use PLATFORM_UNIX define.